--- Make.ACfL	2022-07-19 13:11:08.954180000 -0400
+++ Make.ACfL.patched	2022-07-11 16:45:10.935651000 -0400
@@ -61,13 +61,13 @@
 # - Platform identifier ------------------------------------------------
 # ----------------------------------------------------------------------
 #
-ARCH         = UNKNOWN
+ARCH         = ACfL
 #
 # ----------------------------------------------------------------------
 # - HPL Directory Structure / HPL library ------------------------------
 # ----------------------------------------------------------------------
 #
-TOPdir       = $(HOME)/hpl
+TOPdir       = $(HOME)/benchmarks/hpl-2.3
 INCdir       = $(TOPdir)/include
 BINdir       = $(TOPdir)/bin/$(ARCH)
 LIBdir       = $(TOPdir)/lib/$(ARCH)
@@ -94,7 +94,7 @@
 #
 LAdir        = 
 LAinc        = 
-LAlib        = -lblas
+LAlib        = 
 #
 # ----------------------------------------------------------------------
 # - F77 / C interface --------------------------------------------------
@@ -156,7 +156,7 @@
 #    *) call the BLAS Fortran 77 interface,
 #    *) not display detailed timing information.
 #
-HPL_OPTS     =
+HPL_OPTS     = -DHPL_CALL_CBLAS
 # 
 # ----------------------------------------------------------------------
 #
@@ -167,11 +167,11 @@
 # ----------------------------------------------------------------------
 #
 CC           = mpicc
-CCNOOPT      = $(HPL_DEFS) 
-CCFLAGS      = $(HPL_DEFS) 
+CCNOOPT      = $(HPL_DEFS) -O0
+CCFLAGS      = $(HPL_DEFS) -Ofast -mcpu=neoverse-n1 -armpl
 #
-LINKER       = mpif77
-LINKFLAGS    = 
+LINKER       = mpicc
+LINKFLAGS    = -armpl
 #
 ARCHIVER     = ar
 ARFLAGS      = r
