--- Make.GCC_BLIS	2022-07-19 12:28:40.830337000 -0400
+++ Make.GCC_BLIS.patched	2022-07-19 13:03:47.729051000 -0400
@@ -61,13 +61,13 @@
 # - Platform identifier ------------------------------------------------
 # ----------------------------------------------------------------------
 #
-ARCH         = UNKNOWN
+ARCH         = GCC_BLIS
 #
 # ----------------------------------------------------------------------
 # - HPL Directory Structure / HPL library ------------------------------
 # ----------------------------------------------------------------------
 #
-TOPdir       = $(HOME)/hpl
+TOPdir       = $(HOME)/benchmarks/hpl-2.3
 INCdir       = $(TOPdir)/include
 BINdir       = $(TOPdir)/bin/$(ARCH)
 LIBdir       = $(TOPdir)/lib/$(ARCH)
@@ -92,9 +92,9 @@
 # header files,  LAlib  is defined  to be the name of  the library to be 
 # used. The variable LAdir is only used for defining LAinc and LAlib.
 #
-LAdir        = 
-LAinc        = 
-LAlib        = -lblas
+LAdir        = $(HOME)/benchmarks/blis_gcc-11.2.0_thunderx2
+LAinc        = -I$(LAdir)/include/blis
+LAlib        = -L$(LAdir)/lib -lblis
 #
 # ----------------------------------------------------------------------
 # - F77 / C interface --------------------------------------------------
@@ -156,7 +156,7 @@
 #    *) call the BLAS Fortran 77 interface,
 #    *) not display detailed timing information.
 #
-HPL_OPTS     =
+HPL_OPTS     = -DHPL_CALL_CBLAS
 # 
 # ----------------------------------------------------------------------
 #
@@ -167,10 +167,10 @@
 # ----------------------------------------------------------------------
 #
 CC           = mpicc
-CCNOOPT      = $(HPL_DEFS) 
-CCFLAGS      = $(HPL_DEFS) 
+CCNOOPT      = $(HPL_DEFS) -O0
+CCFLAGS      = $(HPL_DEFS) -Ofast -mcpu=neoverse-n1
 #
-LINKER       = mpif77
+LINKER       = mpicc
 LINKFLAGS    = 
 #
 ARCHIVER     = ar
