--- Make.NVIDIA_HPC_SDK	2022-07-19 13:13:56.121925000 -0400
+++ Make.NVIDIA_HPC_SDK.patched	2022-07-19 13:14:16.536185000 -0400
@@ -61,13 +61,13 @@
 # - Platform identifier ------------------------------------------------
 # ----------------------------------------------------------------------
 #
-ARCH         = UNKNOWN
+ARCH         = NVIDIA_HPC_SDK
 #
 # ----------------------------------------------------------------------
 # - HPL Directory Structure / HPL library ------------------------------
 # ----------------------------------------------------------------------
 #
-TOPdir       = $(HOME)/hpl
+TOPdir       = $(HOME)/benchmarks/hpl-2.3
 INCdir       = $(TOPdir)/include
 BINdir       = $(TOPdir)/bin/$(ARCH)
 LIBdir       = $(TOPdir)/lib/$(ARCH)
@@ -92,9 +92,9 @@
 # header files,  LAlib  is defined  to be the name of  the library to be 
 # used. The variable LAdir is only used for defining LAinc and LAlib.
 #
-LAdir        = 
-LAinc        = 
-LAlib        = -lblas
+LAdir        = $(NVHPC_ROOT)/compilers
+LAinc        = -I$(LAdir)/include
+LAlib        = -L$(LAdir)/lib -lblas
 #
 # ----------------------------------------------------------------------
 # - F77 / C interface --------------------------------------------------
@@ -156,7 +156,7 @@
 #    *) call the BLAS Fortran 77 interface,
 #    *) not display detailed timing information.
 #
-HPL_OPTS     =
+HPL_OPTS     = -DHPL_CALL_CBLAS
 # 
 # ----------------------------------------------------------------------
 #
@@ -167,10 +167,10 @@
 # ----------------------------------------------------------------------
 #
 CC           = mpicc
-CCNOOPT      = $(HPL_DEFS) 
-CCFLAGS      = $(HPL_DEFS) 
+CCNOOPT      = $(HPL_DEFS) -O0 -Kieee
+CCFLAGS      = $(HPL_DEFS) -O3 -fast -Minline=saxpy,sscal -Minfo
 #
-LINKER       = mpif77
+LINKER       = mpicc
 LINKFLAGS    = 
 #
 ARCHIVER     = ar
